dmp/user-story-all-dmps.ipynb

```diff
@@ -134,10 +134,12 @@
    "outputs": [],
    "source": [
     " # Generate the GraphQL query to retrieve up to 100 outputs of University of Oxford, with at least 100 views each.\n",
+    "repo_id = \"cdl.cdl\" if f.value == \"https://ror.org/03yrm5c26\" else \"cern.zenodo\"\n",
+    "funder_id = \"https://doi.org/10.13039/100000141\" if f.value == \"https://ror.org/03yrm5c26\" else \"https://doi.org/10.13039/501100000780\"\n",
```

Comment:
would be nice to allow for a collection of funders here. In particular we have 'NSF' (10.13039/100000001) and a sub-group within NSF, the Division of Ocean Sciences (10.13039/100000141). I didn't see a way to do that below. I tried `funders (containsId: funder_id)` but received errors.